### ISSUE TITLE
torch and CUDA version

### ISSUE THREAD

**Issue by Terencedu** [LinShan-Bin/OccNeRF#20]
Hi,

Thanks for your great work! There are some problems when I installed the torch environment.
My device is Ubuntu 2204, CUDA 11.3, 4 RTX4090 (40 series need at least CUDA 11.3).
My training setting: v1.0-mini dataset. contracted_coord = False, auxiliary_frame = False, render_h = 45, render_w = 80, input_channel = 16, train only depth.

1. `conda install pytorch==1.9.1 torchvision==0.10.1 cudatoolkit=11.3 -c pytorch -c conda-forge`
This is from your repo, but it would install torch of cpu version and torch.cuda.is_available() = False
![1715852887514](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/2f3e581a-f2a1-4025-90d0-e12ea6776d65)

2. ` conda install pytorch==1.10.0 torchvision==0.11.0 torchaudio==0.10.0 cudatoolkit=11.3 -c pytorch -c conda-forge` and
`conda install pytorch==1.10.1 torchvision==0.11.2 torchaudio==0.10.1 cudatoolkit=11.3 -c pytorch -c conda-forge`
These two are from pytorch official website. They are gpu version, but they have the same error.
![截图 2024-05-17 13-47-27](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/fe8df3fc-2053-4de8-8609-05af9a536474)

3. `pip install torch==1.9.1+cu111 torchvision==0.10.1+cu111 torchaudio==0.9.1 -f https://download.pytorch.org/whl/torch_stable.html`
This is from pytorch official website. It trained successfully with contracted_coord = True and little warnings, but  there is error about CUDA with contracted_coord = False, maybe because it is cu111:
![2](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/bc61b8a3-0226-4b33-9e29-76a66d1cf51c)

4. `pip install torch==1.11.0+cu113 torchvision==0.12.0+cu113 torchaudio==0.11.0 --extra-index-url https://download.pytorch.org/whl/cu113` and `conda install pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 cudatoolkit=11.3 -c pytorch`
They both are from pytorch official website. It seems like training successfully. There are lots of warnings about CUDA, is this torch and cuda version suitable? 
![截图 2024-05-17 11-16-26](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/980e0125-45f7-4196-982c-197515c07faf)
![截图 2024-05-17 11-16-36](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/f12dad3d-c8f4-4acd-a2cd-85c902bb99cc)
![截图 2024-05-17 11-16-45](https://github.com/LinShan-Bin/OccNeRF/assets/102672687/a97a5b68-6b5a-4c8f-ad04-274aca872627)

5. How to set the config to train model with semantic with 4090(24GB each)? A little accuracy can be sacrificed. 

Thanks for your help!
